clinica/utils/filemanip.py

```diff
@@ -207,3 +207,40 @@ def extract_metadata_from_json(json_file, list_keys):
         file.close()
 
     return list_values
+
+
+def detect_cross_sectional_and_longitudinal_subjects(
+    all_subs: list, bids_dir: str
+) -> list:
+    """This function detects whether a subject as a longitudinal or cross sectionnal structure.
+    If it has a mixed structure, it will be considered cross sectionnal.
+
+    Parameters
+    ----------
+    all_subs: list
+        List containing all the names of the subjects.
+
+    bids_dir: str
+        Path to the bids directory.
+    Returns
+    -------
+    cross_subj: list
+        List of all the subject detected with a cross_sectionnal organisation.
```

Comment:
```suggestion
        List of all the subject detected with a cross_sectionnal organisation.

```

Reply:
I assumed you wanted me to correct the typo on sectionnal to sectional

Reply:
I just added an empty line to separate the sections, but you're right about the typo that I missed 👍